app/views/events/_invites.html.erb

```diff
@@ -0,0 +1,16 @@
+<ul class="nav navbar-nav navbar-right collapse navbar-collapse" id="bs-example-navbar-collapse-1"">
+  <li class="dropdown">
+      <a href="#" class="dropdown-toggle" data-toggle="dropdown"> 
+       Invitees: <%= User.find(@event.enrollments.invited.map {|en_user| en_user.user_id }).count %> <b class="caret"></b>
+      </a>
+      <% if User.find(@event.enrollments.invited.map {|en_user| en_user.user_id }).any? %>
+      <ul class="dropdown-menu">
+      <% User.find(@event.enrollments.invited.map { |en_user| en_user.user_id }).each do |user| %>

```

Comment:
Here also. There are still a few more in other views. Consider handling that in your controllers or models